src/http/request_handler.rs

```diff
@@ -279,7 +373,11 @@ pub async fn handle_request<T: DeserializeOwned + GraphQLRequestLike>(
     telemetry::propagate_context(&req);
     let mut req_counter = RequestCounter::new(&app_ctx.blueprint.telemetry, &req);
 
-    let response = handle_request_inner::<T>(req, app_ctx, &mut req_counter).await;
+    let response = if let Some(cors_params) = app_ctx.blueprint.server.cors_params.clone() {
```

Comment:
don't clone, unless we need to perform cors.

Reply:
fixed